pkgs/servers/web-apps/outline/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, nodejs
+, yarn
+, yarn2nix-moretea
+}: let
+  pname = "outline";
+  version = "0.65.2";
+
+  src = fetchFromGitHub {
+    owner = "outline";
+    repo = "outline";
+    rev = "v${version}";
+    sha256 = "sha256-a9K6nMgg1j93BPiy03M86dDecXv/J47vUaqHH3S6DOs=";
+  };
+in stdenv.mkDerivation {
+  inherit pname version src;
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ yarn nodejs ];
+
+  # Replace the inline call to yarn with our sequalize wrapper. This should be
+  # the only occurrence:
+  # https://github.com/outline/outline/search?l=TypeScript&q=yarn
+  patches = [ ./sequelize-command.patch ];
+
+  yarnOfflineCache = yarn2nix-moretea.importOfflineCache ./yarn.nix;
+
+  configurePhase = ''
+    export HOME=$PWD/yarn_home
```

Comment:
```suggestion
    export HOME=$(mktemp -d)/yarn_home
```

Reply:
Is there a specific reason why mktemp is better? Yarn2nix does it like this as well:
https://github.com/NixOS/nixpkgs/blob/f6f3787c66e0a1ea4c97ce2647508d5d32f438cd/pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix#L124-L125

Reply:
We don't want to pollute $PWD incase we copy something from there.